templates/model/pos.go

```diff
@@ -62,15 +62,11 @@ func (c *ConfigPos) AnnotateErr(err error) error {
 //	ErrWithPos(pos, "Foo(): %w", err)
 //
 // Calling this function with a zero or nil ConfigPos is valid. The resulting
-// error will just say that the position is unknown.
+// error will just omit information about config location.
 func ErrWithPos(pos *ConfigPos, fmtStr string, args ...any) error {
```

Comment:
Just realized it won't hurt to add a unit test for `ErrWithPos`?

Reply:
Added.